README.md

```diff
@@ -289,6 +289,10 @@ the default protocol to `http-remoting` and the default port to `9990`
 (in case SSL is configured, this switches the default protocol to
 `https-remoting` and the default port to `9993`).
 
+If you want to use HTTP transport for executing operation (`.protocol(ManagementProtocol.HTTP)`),
+you need to provide Apache HttpComponents Client 4 by yourself. Please notice that reload operation through HTTP
+transport may fail with `WildFly 10.0.0`.
```

Comment:
Also, this interrupts the text flow. I'd put this at the end of this chapter, right above the "### Offline" chapter, and into a separate subchapter, "#### HTTP transport".